openassessment/templates/openassessmentblock/peer/oa_peer_assessment.html

```diff
@@ -66,9 +66,11 @@ <h3 class="peer-assessment__display__title">
                                     </h3>
                                 </header>
 
-                                {% include "openassessmentblock/oa_submission_answer.html" with answer=peer_submission.answer answer_text_label="Your peer's response to the question above:" %}
+                                {% trans "Your peer's response to the question above" as transated_label %}
+                                {% include "openassessmentblock/oa_submission_answer.html" with answer=peer_submission.answer answer_text_label=translated_label %}
```

Comment:
Ugh, looks like it's just a typo. `transated_label` vs `translated_label`. I'll get a fix up shortly.